packages/web-app-files/src/App.vue

```diff
@@ -61,7 +65,15 @@ export default defineComponent({
       closeSidebar: 'close',
       setActiveSidebarPanel: 'setActivePanel'
     }),
-
+    onDragEnter(event) {
+      event.stopPropagation()
+      event.preventDefault()
+      const hasFileInEvent = (event.dataTransfer.types || []).some((e) => e === 'Files')
+      this.dragareaEnabled = hasFileInEvent
```

Comment:
Why don't you directly assign `this.dragareaEnabled = (event.dataTransfer.types || []).some((e) => e === 'Files')`?